File: Python Programming/problems on numbericals/problem29.py
'''
29. Accept amount in US dollar and return its corresponding value in Indian currency.
	Consider 1$ as 70 rupees
	
	Author : Milind Nalawade
	
	Date : 28/06/2021
	
	Language : Python 3.9.0

'''
#====
#Logic
#====
def DtoA(no):
	return no*70
#=============
#Entry point function
#=============
def main():
	
	d=int(input("Enter your amount in dollar : "))
	ret=int(DtoA(d))
	print("Amount in rupees is : {} rs".format(ret))
	
#=====
#starter
#=====
if __name__=="__main__":
	main()	

File: C++ Programming/problems on numberical/problem29.cpp
/*

29. Accept amount in US dollar and return its corresponding value in Indian currency.
	Consider 1$ as 70 rupees
	
	Author : Milind Nalawade
	
	Date : 28/06/2021
	
	Language : C++ programming

*/
using namespace std;
#include<iostream>

int DtoR(int no)
{
	return no*70;
}

int main()
{
	int D=0,ret=0;
	cout<<"Enter your amount in dollar : ";
	cin>>D;
	ret=DtoR(D);
	cout<<"amount in rupees : "<<ret<<" rs";
	return 0;
}